chapters/Chapter 1 — Untitled.md

```diff
@@ -39,7 +39,7 @@ Leanne banged a hand against the field generator as she steadied herself with a
 
 Only one person was entering, heavily armed by the looks of him. Leanne was sure there was at least one more staying behind to man their ship’s controls: they’d be flying manually, as their Alts would not be helping them attack her.
 
-Another light flashed; Leanne blinked its echoes from her eyes: the inner airlock doors were opening. Leanne flipped the _Engine Power_ switch, only allowing herself a moment’s pause to relish the roar of the engines engaging. 
+Another blue light flashed; Leanne blinked its echoes from her eyes: the inner airlock doors were opening. Leanne flipped the _Engine Power_ switch, only allowing herself a moment’s pause to relish the roar of the engines engaging. 
```

Comment:
Not sure how well "blue" works here; perhaps if I work in some other mentions of light colors?